aospy/test/test_region.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from aospy import Region
+from aospy.region import _get_land_mask
+from aospy.internal_names import (LAT_STR, LON_STR,
+                                  SFC_AREA_STR, LAND_MASK_STR)
+
+
+@pytest.fixture()
+def data():
+    lat = [-10., 1., 10., 20.]
+    lon = [1., 10.]
+    sfc_area = [0.5, 1., 0.5, 0.25]
+    land_mask = [1., 1., 0., 1.]
+
+    lat = xr.DataArray(lat, dims=[LAT_STR], coords=[lat])
+    lon = xr.DataArray(lon, dims=[LON_STR], coords=[lon])
+    sfc_area = xr.DataArray(sfc_area, dims=[LAT_STR], coords=[lat])
+    land_mask = xr.DataArray(land_mask, dims=[LAT_STR], coords=[lat])
+
+    sfc_area, _ = xr.broadcast(sfc_area, lon)
+    land_mask, _ = xr.broadcast(land_mask, lon)
+
+    da = xr.DataArray([[2., 2.],
+                       [np.nan, 5.],
+                       [3., 3.],
+                       [4., 4.]], coords=[lat, lon])
+    da[SFC_AREA_STR] = sfc_area
+    da[LAND_MASK_STR] = land_mask
+    return da
+
+
+@pytest.fixture()
+def region_no_land_mask():
+    return Region(
+        name='test',
+        description='Test region with no land mask',
+        lat_bounds=(0., 90.),
+        lon_bounds=(0., 5.),
+        do_land_mask=False
+    )
+
+
+@pytest.fixture()
+def region_land_mask():
+    return Region(
+        name='test',
+        description='Test region with no land mask',
+        lat_bounds=(0., 90.),
+        lon_bounds=(0., 5.),
+        do_land_mask=True
+    )
+
+
+def test_mask_var(data, region_no_land_mask):
+    # Test region masks first row and second column
+    # of test data.  Note that first element of
+    # second row is np.nan in initial dataset.
+    expected_data = [[np.nan, np.nan],
+                     [np.nan, np.nan],
+                     [3., np.nan],
+                     [4., np.nan]]
+    expected = data.copy(deep=True)
+    expected.values = expected_data
+
+    result = region_no_land_mask.mask_var(data)
+    xr.testing.assert_identical(result, expected)
+
+
+def test_get_land_mask_without_land_mask(data, region_no_land_mask):
+    result = _get_land_mask(data, region_no_land_mask.do_land_mask)
+    expected = 1
+    assert result == expected
+
+
+def test_get_land_mask_with_land_mask(data, region_land_mask):
+    result = _get_land_mask(data, region_land_mask.do_land_mask)
+    expected = data[LAND_MASK_STR]
+    xr.testing.assert_identical(result, expected)
+
+
+def test_ts_no_land_mask(data, region_no_land_mask):
+    result = region_no_land_mask.ts(data)
+    expected = xr.DataArray((0.25 * 4. + 0.5 * 3.) / (0.25 + 0.5))
```

Comment:
Can you add a note explaining where these values come from?  And/or better yet pull them directly from the test data.